package.json

```diff
@@ -34,14 +34,14 @@
         "concat-stream": "2.0.0",
         "cross-env": "6.0.3",
         "faker": "4.1.0",
-        "jest": "25.5.4",
+        "jest": "27.0.6",
         "jest-extended": "0.11.5",
         "nock": "12.0.3",
         "prettier": "1.19.1",
         "pryjs": "1.0.3",
         "rimraf": "3.0.2",
         "rosie": "2.0.1",
-        "ts-jest": "26.4.1",
+        "ts-jest": "27.0.3",
```

Comment:
Huh. I thought we were blocked from newer jest/ts-jest versions because of node 8. Neat 👍

Reply:
Hmmm....  I may not have been running under node 8.  I guess I'd have to ask is whether we need to be on node 8 or can we bump it up?

Reply:
Yeah, I pulled it down locally and switched to Node 8 and the tests won't run:

```
Brandon@din ~/acc/fork/myty.AndcultureCode.Cli/ (feature/add-dotnet-watch-test) -> nvm use 8
Now using node v8.16.2 (npm v6.4.1)
Brandon@din ~/acc/fork/myty.AndcultureCode.Cli/ (feature/add-dotnet-watch-test) -> npm t

> and-cli@2.8.0 test /Users/Brandon/acc/fork/myty.AndcultureCode.Cli
> jest

/Users/Brandon/acc/fork/myty.AndcultureCode.Cli/node_modules/jest/node_modules/jest-cli/build/cli/index.js:227
    } catch {
            ^

SyntaxError: Unexpected token {
    at createScript (vm.js:80:10)
    at Object.runInThisContext (vm.js:139:10)
    at Module._compile (module.js:617:28)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/Brandon/acc/fork/myty.AndcultureCode.Cli/node_modules/jest/node_modules/jest-cli/bin/jest.js:16:3)
npm ERR! Test failed.  See above for more details
```

At this point we can probably upgrade the minimum engine requirement to 14 since the team has upgraded.(See #162). We've been running Node v14 builds for a while now and the tests have been consistently passing, so I'm not concerned with a big risk.

As for why the CI builds for this PR passed... I'll have to investigate this further. My hunch is that it's running tests for code on the main branch, not the branch from the PR 🤦 that would line up with why codecov has been reporting 0% coverage differences in the recent PRs.